report/json.go

```diff
@@ -5,22 +5,15 @@ import (
 	"io"
 )
 
-func writeJson(findings []Finding, w io.WriteCloser) error {
-	if len(findings) == 0 {
-		findings = []Finding{}
-	}
-	for i := range findings {
-		// Remove `Line` from JSON output
-		findings[i].Line = ""
-	}
-	return writeJsonExtra(findings, w)
+type JsonReporter struct {
 }
 
-func writeJsonExtra(findings []Finding, w io.WriteCloser) error {
+var _ Reporter = (*JsonReporter)(nil)
```

Comment:
@rgmz is this injection necessary?

Reply:
This provides a compile-time check that `JsonReporter` satisfies `Reporter`, right? Or are you referring to something different?

Reply:
nah, that's what I'm referring to. It's not necessary but I can see the value it provides